Phonebook.Frontend/src/app/pages/settings/settings.component.html

```diff
@@ -41,17 +42,15 @@ <h2 i18n="SettingsComponent|Color Theme SubTitle@@SettingsComponentSubTitleColor
           </mat-form-field>
         </mat-list-item>
       </mat-list>
-      <h2 i18n="SettingsComponent|Layout Subtitle@@SettingsComponentSubTitleLayout">Layout</h2>
+      <mat-card-subtitle class="subtitleSetting" i18n="SettingsComponent|Subtitle Dashboard of the settings page@@SettingsComponentSubtitleDashboard">
+        Dashboard</mat-card-subtitle>
+      <h2 i18n="SettingsComponent|Layout Subtitle@@SettingsComponentSubTitleLayout">Layout for Bookmarks</h2>
       <mat-list>
         <mat-list-item>
-          <mat-icon class="pb-margin-20">{{ layoutValue }}</mat-icon>
-          <mat-form-field>
-            <mat-select [(ngModel)]="layoutValue" (ngModelChange)="changeLayout(layoutValue)">
-              <mat-select-trigger>
-                <span>{{ getLayoutName(layoutValue) }}</span>
-              </mat-select-trigger>
-              <mat-option *ngFor="let view of layout" [value]="view">
-                <mat-icon>{{ view }}</mat-icon>
+          <mat-icon class="pb-margin-20">view_list</mat-icon>
+            <mat-form-field>
+              <mat-select [ngModel]="layoutValue$ | async" (ngModelChange)="changeLayout($event)">
+                <mat-option *ngFor="let view of layouts" [value]="view">
```

Comment:
Can you change the order of the list? (By changing the order in the enum) So that it is ordered from biggest to smallest and not "smallest". "biggest", "medium"